__tests__/app.test.js

```diff
@@ -156,4 +156,50 @@ describe("/api/articles", () => {
             });
     });
  //should I test for inaccurate count? not a 400 or 404, end user couldn't know   
+})
+describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: Responds with endpoint json data", () => {
+        return request(app)
+            .get("/api/articles/3/comments")
+            .expect(200)
+            .then(({ body }) => {
+              const comments = body.comments;
+              comments.forEach((comment) => {
+                  expect(typeof comment.comment_id).toBe("number");
+                  expect(typeof comment.votes).toBe("number");
+                  expect(typeof comment.created_at).toBe("string");
+                  expect(typeof comment.author).toBe("string");
+                  expect(typeof comment.body).toBe("string");
+                  expect(typeof comment.article_id).toBe("number");
```

Comment:
On the path, you've looked at article 3, right? So it's safe to say that every comment should have an article_id of 3 :)